parsing/dialects/dialect1/src/test/java/org/apache/calcite/test/Dialect1ParserTest.java

```diff
@@ -2695,7 +2695,7 @@ private void checkExNotNull(SqlParserUtil.StringAndPos sap, Throwable thrown) {
 
   @Test public void testPeriodTypeNameSpecTime() {
     final String sql = "create table foo (a period(time))";
-    final String expected = "CREATE TABLE `FOO` (`A` PERIOD(TIME))";
+    final String expected = "CREATE TABLE `FOO` (`A` PERIOD(TIME(6)))";
```

Comment:
To be on the safe side, I'd recommend we unparse without any precision if no precision was specified in the input query.

Reply:
Updated on https://github.com/googleinterns/calcite/pull/177/commits/07868fb1af151e9941c0be5c15dc4cb9e0d57e2f